cropharvest/eo/eo.py

```diff
@@ -55,8 +57,23 @@ def __init__(self, data_folder: Path = DATAFOLDER_PATH) -> None:
         # allows for easy checkpointing
         self.cur_output_folder = f"{self.output_folder_name}_{str(date.today()).replace('-', '')}"
 
-        self.labels = geopandas.read_file(data_folder / LABELS_FILENAME)
-        self.labels["export_end_date"] = pd.to_datetime(self.labels["export_end_date"])
+        self.labels = self.default_labels if labels is None else labels
+        self.using_default = labels is None
+        for expected_column in ["start_date", "end_date", "lat", "lon"]:
+            assert expected_column in self.labels
+        if "export_identifier" not in self.labels:
+            print("No explicit export_identifier in labels. One will be constructed during export")
+
+    @property
+    def default_labels(self) -> geopandas.GeoDataFrame:
+        labels = geopandas.read_file(self.data_folder / LABELS_FILENAME)
+        export_end_year = pd.to_datetime(self.labels["export_end_date"]).dt.year
```

Comment:
How do "export_end_date" and "end_date" differ?

Reply:
In this case, they don't. I implemented it this way because
a) it keeps the naming consistent (as per the issue) 
b) it makes sure (for the default labels) the right months / dates are exported

Reply:
So is `export_end_date` a default column for all labels? If yes, I can just rename the column on my end. The column names don't have to be strict just consistent

Reply:
I actually don't mind having it this way, since it is an additional check in the exporter that the right date is exported. I'll update the function to use the Column object though.